app/data/locales.json

```diff
@@ -82,5 +82,12 @@
     "value": "sv",
     "key": "i18n.lang.sv",
     "level": 1
-  }
+  },
+  {
+    "label": "Русский",
+    "name": "Russian",
+    "value": "ru",
```

Comment:
Do these translations resemble Russian from Russia (ru_RU), from another country (such as ru_UA), or general Russian targeted to all Russian speakers (ru)?